packages/ferreiro-server/setup/setupMiddlewares.js

```diff
@@ -7,6 +7,15 @@ const helmet = require('helmet')
 const session = require('express-session')
 const express = require('express')
 const compression = require('compression')
+const rateLimit = require('express-rate-limit');
+
+const crawlersRateLimit = rateLimit({
+  windowMs: 15 * 60 * 1000, // 15 minutes
+  max: 15 // limit each IP to 15 requests per windowMs
+});
+
+// TODO: Move to it's own repository and deploy...
+const sitemapGenerator = require('../../fancy-sitemap/index')
```

Comment:
You are in a workspace and it is defined as a package. Let the workspaces handle resolution.